src/common.build.post.props

```diff
@@ -30,6 +30,7 @@
   <PropertyGroup>
     <CAExcludePath>$(SolutionDir)\dep\;$(CAExcludePath)</CAExcludePath>
   </PropertyGroup>
+  <!--

```

Comment:
Errr.... you have further changes to this, right?

Reply:
Yeah, I'll comment (and discuss) this in the PR I split off this one to fix the build.